addons/website_slides_survey/views/survey_survey_views.xml

```diff
@@ -25,4 +25,21 @@
             </p>
         </field>
     </record>
+
+    <record id='survey_survey_view_form' model="ir.ui.view">
+        <field name="name">survey.survey.view.form</field>
```

Comment:
As you inherit in the website_slides context, it will be `<field name="name">survey.survey.view.form.inherit.website_slides</field>` here.

Reply:
Fixed it